### PR TITLE
Day2

### PR DESCRIPTION
**Pull request by Jyothi-Sittula**
Day2 Pull

### CODE REVIEW

File: PlusMinus.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            //(num==0)?ratios[2]+=1:( (num>0)?ratios[0]+=1:ratios[1]+=1);
+            //error CS0201
+            if(num==0)
+                ratios[2]++;
+            else{
+                if(num>0)
+                    ratios[0]++;
+                else   
+                    ratios[1]++;
+            }
```
Comment (narayanagunturi) [inline PlusMinus.cs]: refractor this code to use else if instead of nested if, else statements @Swaroop-Sittula

File: StairCase.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+     */
+
+    public static void staircase(int n)
+    {
+        for(var i=1;i<=n;i++)
+        {
+           for(var j=1;j<=n-i;j++)
+            Console.Write(' ');
+           for(var j=1;j<=i;j++)
+            Console.Write('#');
```
Comment (narayanagunturi) [inline StairCase.cs]: Please maintain consistency everywhere. You are using 4 spaces in the above lines. But you are using 2 spaces here @Swaroop-Sittula

File: PlusMinus.cs (hunk 1 above)
Comment (Jyothi-Sittula) [reply PlusMinus.cs]: Updated the code @narayanagunturi

File: StairCase.cs (hunk 2 above)
Comment (Jyothi-Sittula) [reply StairCase.cs]: Okay @narayanagunturi

**Review by narayanagunturi — APPROVED** [review]

**Pull request merged**